Formula/go@1.9.rb

```diff
@@ -0,0 +1,95 @@
+class GoAT19 < Formula
+  desc "Go programming environment (1.9)"
+  homepage "https://golang.org"
+
+  stable do
```

Comment:
As there is no head block anymore, `stable do` should be removed and the content of the block left-shifted one level.